torch_geometric/nn/models/lightgcn.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+import torch
+from torch import nn, Tensor
+from torch_geometric.typing import Adj
+
+from torch_geometric.nn.conv import LightGCNConv
+
+
+class LightGCN(nn.Module):
+    """ LightGCN implementation in PyG. From :
+    "LightGCN: Simplifying and Powering Graph Convolution Network
+    for Recommendation"
+    Adapted from the original author's repo:
+    https://github.com/gusye1234/LightGCN-PyTorch
+    """
+    def __init__(self,
+                 config: dict,
+                 device=None,
+                 **kwargs):
+        super().__init__()
+
+        self.num_users = config["n_users"]
+        self.num_items = config["m_items"]
+        self.embedding_size = config["embedding_size"]
+        self.in_channels = self.embedding_size
+        self.out_channels = self.embedding_size
+        self.num_layers = config["num_layers"]
+
+        # 0-th layer embedding.
+        self.embedding_user_item = torch.nn.Embedding(
+            num_embeddings=self.num_users + self.num_items,
+            embedding_dim=self.embedding_size)
+        self.alpha = None
+
+        # random normal init seems to be a better choice
+        # when lightGCN actually don't use any non-linear activation function
+        nn.init.normal_(self.embedding_user_item.weight, std=0.1)
+        print('use NORMAL distribution initilizer')
+
+        self.f = nn.Sigmoid()
+
+        self.convs = nn.ModuleList()
+        self.convs.append(LightGCNConv(
+            self.embedding_size, self.embedding_size,
+            num_users=self.num_users, num_items=self.num_items, **kwargs))
+
+        for _ in range(1, self.num_layers):
+            self.convs.append(LightGCNConv(
+                self.embedding_size, self.embedding_size,
+                num_users=self.num_users, num_items=self.num_items, **kwargs))
+
+        self.device = None
+        if device is not None:
+            self.convs.to(device)
+            self.device = device
+
+    def reset_parameters(self):
+        for conv in self.convs:
+            conv.reset_parameters()
+
+    def forward(self, x: Tensor, edge_index: Adj, *args, **kwargs) -> Tensor:
+        """ """
+        xs: List[Tensor] = []
```

Comment:
```suggestion
        xs: List[Tensor] = [x]
```
The initial embedding is needed as well.